Exiled.API/Features/Player.cs

```diff
@@ -1666,18 +1666,14 @@ public void TrySetCustomRoleFriendlyFire(Dictionary<string, Dictionary<RoleTypeI
         /// <summary>
         /// Forces the player to reload their current weapon.
         /// </summary>
-        /// <exception cref="InvalidOperationException">If the item is not a firearm.</exception>
-        public void ReloadWeapon() // TODO: Convert to bool instead of Exception
+        /// <returns>True if succesful, otherwise false.</returns>
```

Comment:
spelling error

Reply:
Nao skill issue :trolling:

Reply:
Mine? I said `<returns><see langword="true"/> if the weapon was reloaded; otherwise, <see langword="false"/>.</returns>` lol

Reply:
> Mine? I said `<returns><see langword="true"/> if the weapon was reloaded; otherwise, <see langword="false"/>.</returns>` lol
> 

Bruh i have thinks he have copied your lol i didnnt verified. ^^"